src/Routing/Route/Route.php

```diff
@@ -423,6 +423,7 @@ public function match(array $url, array $context = [])
             $hostOptions['_ext'] = $url['_ext'];
             unset($url['_ext']);
         }
+        unset($defaults['_ext']);
```

Comment:
The paste you added doesn't make sense to me. The 2nd param of scope() is for defining static parameters, not patterns.